internal/envconfig/envconfig.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/go-logr/logr"
 )
 
-// Returns the string value of environment variable "key" or the default value
+// GetEnvString Returns the string value of environment variable "key" or the default value
```

Comment:
If we update the function doc strings, we should make sure they read correctly, i.e. lower case the word "Returns" here. Same for the changed doc strings below.

Reply:
Cool, I'll revert these doc changes and, instead, opened https://github.com/kserve/modelmesh-runtime-adapter/issues/72